submission/Vagrantfile

```diff
@@ -6,8 +6,8 @@ Vagrant.configure("2") do |config|
     end
 
     config.nfs.functional = false
-    config.vm.provision 'shell', inline: 'mv /vagrant/submission.zip .; \
-                                            unzip submission.zip; \
+    config.vm.provision 'shell', inline: 'mv /vagrant/archive.zip .; \
+                                            unzip archive.zip; \
                                             chown -R vagrant:vagrant .'
```

Comment:
Hmm, why don't you run things in the `/vagrant` folder, since they are already rsync'ed there? That's the vagrant way. So you can remove this `mv` and `chown` entirely. And it should be the responsibility of `submission.sh` to unpack the zip.